bin/i18n/sync-all.rb

```diff
@@ -130,12 +131,7 @@ def parse_options(args)
       opts.on("-y", "--yes", "Run without confirmation") do
         options[:yes] = true
       end
```

Comment:
I wonder if we should move the rest of the options to the one parser. On one hand, having one parsing method could be cleaner, on the other hand, most options are only used in the `sync-all` script, and could get confusing to have options available that are not used in other modules.

Reply:
We shouldn't add options that we are not going to use them in the scripts